chain-extensions/types/dapps-staking/src/lib.rs

```diff
@@ -120,27 +123,29 @@ pub enum Contract<Account> {
     Wasm(Account),
 }
 
+pub type ContractBytes = [u8; 32];
```

Comment:
Since this type is in public it would be a good idea to add some explanatory comment. What is this type, why it 32 bytes long, what is the purpose of that type, etc.